exp/internal/controllers/machinepool_controller.go

```diff
@@ -383,3 +383,22 @@ func (r *MachinePoolReconciler) nodeToMachinePool(ctx context.Context, o client.
 
 	return nil
 }
+
```

Comment:
Nit: let's move this to machinepool_controller_phases.go (because it is only used there)